Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -623,10 +623,10 @@ static WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     m_backgroundWebView.get().inspectable = inspectable;
 
     for (auto entry : m_extensionPageTabMap)
-        entry.key.cocoaView().get().inspectable = inspectable;
+        Ref { entry.key }->cocoaView().get().inspectable = inspectable;
```

Comment:
Eh, this looks bad. Can you put in a local var?

Reply:
Good point--fixed!